crates/forge/src/test_case_summary.rs

```diff
@@ -41,17 +41,7 @@ pub enum TestCaseSummary {
         name: String,
     },
     /// Test case skipped due to exit first
```

Comment:
Is this only possible reason for `Skipped` status now?

Reply:
In general, it's returned when we do not care about the test result, in the case when there is no test that returns Err we use it to know how many tests was skipped

Reply:
I'd update this comment then